<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadMainWindow</class>
 <widget class="QWidget" name="UploadMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>276</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>522</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>785</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>上传文件</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="baseWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#baseWidget {background-color: rgb(255, 255, 255);}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="4" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" rowspan="2">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>17</width>
          <height>58</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>141</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="browseComputerBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>183</width>
             <height>39</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>183</width>
             <height>39</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">#browseComputerBtn {background-color: rgb(26, 115, 232);
font: 14pt &quot;PingFang SC&quot;;
font: 14pt &quot;Alibaba PuHuiTi 2.0 55 Regular&quot;;
color: rgb(255, 255, 255);
font: 14pt &quot;SimHei&quot;;
border-width: 2px;
border-radius: 8px;
}
</string>
           </property>
           <property name="text">
            <string>浏览您的计算机</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QStackedWidget" name="uploadStackedWidget">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">#uploadStackedWidget {
border: 2px dotted rgb(0, 0, 0);
border-radius: 8px;
width: 40%;}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="stackedWidgetPage1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>33</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="chooseFileBtn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#chooseFileBtn{border: 0px;
font: 700 16pt &quot;Microsoft YaHei UI&quot;;}</string>
            </property>
            <property name="text">
             <string>选择您的文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uploadCsvBtn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#uploadCsvBtn{border: 0px;
font: 10pt &quot;Microsoft YaHei UI&quot;;}</string>
            </property>
            <property name="text">
             <string>请上传.csv 文件</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page">
         <widget class="QWidget" name="fileIconWidget" native="true">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>20</y>
            <width>71</width>
            <height>71</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#fileIconWidget{border-image: url(:/icon/icon/csv-64.png)}</string>
          </property>
         </widget>
         <widget class="QLabel" name="uploadCompleteLabel">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>60</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#uploadCompleteLabel{font: 14pt &quot;PingFang SC&quot;;}</string>
          </property>
          <property name="text">
           <string>文件上传成功!</string>
          </property>
         </widget>
         <widget class="QProgressBar" name="fileUploadProgressBar">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>100</y>
            <width>300</width>
            <height>16</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
#fileUploadProgressBar::chunk{
	background: rgb(189, 189, 189)
}</string>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
         <widget class="QLabel" name="fileNameLabel">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>40</y>
            <width>241</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">#fileNameLabel{font: 11pt &quot;PingFang SC&quot;;}</string>
          </property>
          <property name="text">
           <string>abc.csv</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chooseFileBtn</sender>
   <signal>clicked()</signal>
   <receiver>browseComputerBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadCsvBtn</sender>
   <signal>clicked()</signal>
   <receiver>browseComputerBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browse_computer_btn_clicked()</slot>
 </slots>
</ui>
